src/acceptance/template/template.go

```diff
@@ -0,0 +1,74 @@
+package template
+
+import (
+	"bytes"
+	"io/ioutil"
+	"strconv"
+)
+
+// Parameter names to replace them to a value.
+var (
+	EndTimeValue   = "{endTimeValue}"
+	MaxCount       = "{maxCount}"
+	StartDateValue = "{startDateValue}"
+	StartTimeValue = "{startTimeValue}"
+	ReportInterval = "{reportInterval}"
+)
+
+var defaultValuesForPolicy = map[string]string{
+	MaxCount:       "5",
+	ReportInterval: "120",
+	StartTimeValue: "00:00",
+	StartDateValue: "2015-06-19",
+	EndTimeValue:   "08:00",
+}
+
+func GeneratePolicy(template_file string, values map[string]string) ([]byte, error) {
+	template, err := ioutil.ReadFile(template_file)
+	if err != nil {
+		return nil, err
+	}
+
+	merged_values := mergeValuesAndDefaultValues(values)
+
+	for k, v := range merged_values {
+		template = bytes.Replace(template, []byte(k), []byte(v), -1)
+	}
+
+	return template, nil
+}
+
+func mergeValuesAndDefaultValues(values map[string]string) map[string]string {
+	var merge_values = map[string]string{}
+	if values == nil {
+		return defaultValuesForPolicy
+	}
+
+	for k, default_v := range defaultValuesForPolicy {
+		value, ok := values[k]
+		if ok {
+			merge_values[k] = value
+		} else {
+			merge_values[k] = default_v
+		}
+	}
+
+	return merge_values
+}
+
+func setValue(key string, value string, values map[string]string) map[string]string {
```

Comment:
Why not just create a type called TemplateValues which is
type TemplateValues map[string]string

Then have 2 funcs on it which are SetString and SetInt.